### ISSUE TITLE
Basemap selector conflicting with other tools in mobile

### ISSUE THREAD

**Issue by ryanbranciforte** [trailheadlabs/outerspatial-map-library#69]
![img_0818](https://user-images.githubusercontent.com/2602563/31631685-47c70342-b26f-11e7-925c-af89e52f7c36.PNG)

Related to this ticket - https://github.com/trailheadlabs/outerspatial-map-library/issues/57

**Comment by ryanjhodge** [comment]
What iPhone version was this on? This is the custom sherif's meadow map? From what I can test in chrome the length of the search control looks okay on iPhone version >= 6plus. Should we try and fix it on the app side or should we support below iPhone 6plus.

What is making the overlap even worse is the side padding/margin around the map on the website itself, which is not related to map library. So I think we should try and fix this on the app side.

**Comment by nateirwin** [comment]
I agree we should take a look at this in the SMF site before spending too much time on it. Here's the SMF website repo: https://github.com/trailheadlabs/sheriffsmeadow-wp-theme.

**Comment by ryanbranciforte** [comment]
FWIW, I think the side padding on the phone was so that users can still scroll up/down the page. When the map is at full screen  width, you are stuck in the map and can't scroll up or down since it's using touch controls when you are on the map.

**Comment by ryanbranciforte** [comment]
So that side padding was intentional

**Comment by ryanbranciforte** [comment]
@ryanjhodge I think I took that first screenshot on my iPhone 7

**Comment by ryanjhodge** [comment]
@nateirwin and I took a look at this and we figured that there are not really any "good" solutions. We recommend just removing the the base layer switcher from this map.  Removing the base layer switcher does not greatly reduce the utility of the map as the map's purpose is for exploring and discovering SMF recreation resources and not for navigation or planning (as iOS is).

Once we redo the basemap switcher we can revisit adding it back in.
